### ISSUE TITLE
[OMP] pkpModalConfirmButton is inactive

### ISSUE THREAD

**Issue by crism** [pkp/pkp-lib#1079]
For instance, at `.../index.php/test/reviewer/submission/2`, a reviewer sees a link, “About Due Dates.” This pops up a modal dialog with an “OK” button. Clicking the button does nothing, though the X in the corner works to dismiss the dialog (as `pkpModalCloseButton`).

Something seems to go wrong with setting the handler in `ButtonConfirmModalHandler.js`; the Firefox inspector shows that the OK button has no handlers (while the X does), but I can’t figure out what the problem is.

**Comment by asmecher** [comment]
Confirmed -- I'll take a look.

**Comment by asmecher** [comment]
The above commit is a partial fix -- but see https://github.com/pkp/pkp-lib/commit/9d2c254da951af4ccbc8080d20c58cd802c53020#commitcomment-15620509

**Comment by NateWr** [comment]
@asmecher I issued a PR with a fix almost identical to yours. I'd like to keep the event handler registration out of `modalBuild`, so that that function does nothing but construct the modal's HTML. Just to keep it's scope small.

Instead I added the event registration to the constructor function in ConformationModalHandler, which is where it should have been.

I don't think the user of `outerHTML()` is related, as mentioned [here](https://github.com/pkp/pkp-lib/commit/9d2c254da951af4ccbc8080d20c58cd802c53020#commitcomment-15620509). My tests showed no problems once the handler was bound to `ConfirmationModalHandler.js`.

Thanks for reporting this @crism!
